contents/mac/create_bin_folder.tw2

```diff
@@ -35,10 +35,10 @@ Your terminal should reply with the following message when the folder was not cr
 
 ## Step 2
 
-Once the folder was created, paste in the following command in your terminal to move the BINARY_NAME program to the `bin` folder:
+Once the folder was created, paste in the following command in your terminal to move the BINARY_NAME program to the `~/bin` folder:
 
 ```
-mv ~/Downloads/BINARY_NAME ~/bin
+mv ~/Downloads/exercism-3.0-*/BINARY_NAME/exercism ~/bin
```

Comment:
```suggestion
mv ~/Downloads/exercism-3.0-*/BINARY_NAME/ ~/bin
```
I'm pretty sure the extra `exercism` isn't necessary

Reply:
Depends on how the extraction works.  I can't confirm, as I don't have the MacOSX tooling. I do know that at the moment, the internal file to the directory is in fact `exercism` as I have seen that in action.